.github/workflows/release-nym-vpn-desktop.yml

```diff
@@ -247,7 +247,7 @@ jobs:
       - name: Generate build info (${{ env.PLATFORM_ARCH }})
         if: contains(matrix.os, 'ubuntu-22.04')
         run: |
-          echo ${{ steps.package-version.outputs.metadata }} > build-info.txt
+          ./nym-vpn-desktop/src-tauri/target/release/bundle/appimage/nym-vpn_*-dev_amd64.AppImage --build-info > build-info.txt
```

Comment:
use `nym-vpn*.AppImage --build-info` to avoid any future name mismatch?

Reply:
Good catch, yeah the dev in particular shouldn't be there